apps.yaml

```diff
@@ -251,35 +251,35 @@ HierCVAE:
         state: development
         title: HierCVAE
         external_url: https://biolib.com/ATGGAGCTGTACGTTAAG/HierCVAE/
-# yadg:
-#     metadata:
-#         title: yadg
-#         description: |
-#             yadg: yet another datagram. FAIR data parser for time-resolved experimental data. Part of the dgbowl suite of tools for digital (electro-)catalysis and battery materials research, developed at Empa.
-#         authors: Peter Kraus, Nicolas Vetsch
-#         documentation_url: https://dgbowl.github.io/yadg
-#         logo: https://dgbowl.github.io/images/yadg.png
-#         state: development
-#         version: '4.1'
-#     git_url: https://github.com/dgbowl/yadg
-#     categories:
-#         - technology-lab
-#         - stakeholder
-#         - stakeholder-aurora
-# tomato:
-#     metadata:
-#         title: tomato
-#         description: |
-#             tomato: au-tomation without the pain. Instrument automation library. Part of the dgbowl suite of tools for digital (electro-)catalysis and battery materials research, developed at Empa.
-#         authors: Peter Kraus
-#         documentation_url: https://dgbowl.github.io/tomato
-#         logo: https://dgbowl.github.io/images/tomato.png
-#         state: development
-#     git_url: https://github.com/dgbowl/tomato
-#     categories:
-#         - technology-lab
-#         - stakeholder
-#         - stakeholder-aurora
+yadg:
+    metadata:
+        title: yadg
+        description: |
+            yadg: yet another datagram. FAIR data parser for time-resolved experimental data. Part of the dgbowl suite of tools for digital (electro-)catalysis and battery materials research, developed at Empa.
+        authors: Peter Kraus, Nicolas Vetsch
+        documentation_url: https://dgbowl.github.io/yadg
+        logo: https://dgbowl.github.io/images/yadg.png
+        state: development
+        version: '4.2'
+    git_url: https://github.com/dgbowl/yadg
+    categories:
+        - technology-lab
+        - stakeholder
+        - stakeholder-aurora
+tomato:
+    metadata:
+        title: tomato
+        description: |
+            tomato: au-tomation without the pain. Instrument automation library. Part of the dgbowl suite of tools for digital (electro-)catalysis and battery materials research, developed at Empa.
+        authors: Peter Kraus, Loris Ercole
+        documentation_url: https://dgbowl.github.io/tomato
+        logo: https://dgbowl.github.io/images/tomato.png
+        state: development
+        version: '0.2'
+    git_url: https://github.com/dgbowl/tomato
+    categories:
+        - technology-lab - stakeholder
```

Comment:
There should be a line break between `-technology-lab` and `-stakeholder`. Thank you for making this change.